x-pack/test/apm_api_integration/tests/service_overview/instances_main_statistics.spec.ts

```diff
@@ -418,6 +418,15 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             expect(javaStats).to.be(undefined);
           });
 
+          it('does not return metrics', () => {
+            const goAStats = body.currentPeriod.find(
+              (stat) => stat.serviceNodeName === 'go-instance-a'
+            );
+
+            expect(goAStats?.memoryUsage).to.be(undefined);
+            expect(goAStats?.cpuUsage).to.be(undefined);
```

Comment:
these 2 expects will still hold true if `goAStats` itself goes undefined. If we want to test the `memoryUsage` and `cpuUsage` for `undefined` explicitly, you should remove the optional chaining.

Reply:
good point!

I still need to use the optional chaining cause the return type can be undefined but adding `expect(goAStats).to.not.be(undefined);` should explicitly check the memory and cpu 

https://github.com/elastic/kibana/pull/145348/commits/87fa356225ea250321f2c8727f4d79db3595a423